src/handlers/int-test-support/helpers/query-athena-helper.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  getQueryExecutionStatus,
+  startQueryExecutionCommand,
+  getQueryResults,
+} from "./athenaHelper";
+import { resourcePrefix } from "./envHelper";
+import { queryAthena } from "./queryHelper";
+
+jest.mock("./athenaHelper", () => {
+  const original = jest.requireActual("./athenaHelper");
+  return {
+    ...original,
+    getQueryExecutionStatus: jest.fn(),
+    startQueryExecutionCommand: jest.fn(),
+    getQueryResults: jest.fn(),
+  };
+});
+const mockedGetQueryExecutionStatus = getQueryExecutionStatus as jest.Mock;
+const mockedStartQueryExecutionCommand =
+  startQueryExecutionCommand as jest.Mock;
+const mockedGetQueryResults = getQueryResults as jest.Mock;
+
+const expectedResults = [{ id: 1, name: test }];
+
+describe("queryAthena", () => {
+  const prefix = resourcePrefix();
+  const databaseName = `${prefix}-calculations`;
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it("should execute a query and return the results", async () => {
+    jest.setTimeout(35000);
```

Comment:
the general jest timeout for unit test looks 5000 ms but the queryAthena function has poll timeout of 30000 so added the timeout for these tests